//= error code 2

fn main() {
    std::exit(2i32);
}